macros/run_end/save_results_history.sql

```diff
@@ -98,11 +98,12 @@
 
 {% endmacro %}
 
-{% macro priv_full_name_from_depends(node, name) %}
+{% macro priv_full_name_from_depends(node, name, type) %}
 
     {% for full_name in node.depends_on.nodes %}
         {% set node_name = full_name.split('.')[-1] %}
-        {% if node_name == name %}
+        {% set node_type = full_name.split('.')[0] %}
```

Comment:
Would that work for `seed` node type?

Reply:
Not sure actually, didn't have any seed data in our project so never tested it. Would that come as a separate type here or just as a source or something? I can have a play with seed data if you are not aware and update the PR if needed.

Reply:
@akshaykarle Not 100% sure if `seeds` are a separate type. If you could play with it this would be great, we also have seeds in our tests so those would likely detect that. (We should add the ability to run those on PR's external contributors, as it's a bit hard to test this now)

Reply:
thanks, I will test it then and update here in the next few days. and yes, if the tests could run on PR's that would be amazing.

Reply:
Got it, will look into running them ;)

Reply:
hey @mateuszklimek I finally got some time to test out this issue with seed data. It looks like we cannot have this issue with seed data since we specifically pull only refs and sources based on the regex - https://github.com/re-data/dbt-re-data/blob/6b04f3f4643f458131b836727b64161e2882e144/macros/run_end/save_results_history.sql#L38. I did also test adding a seed node with the same name as the model but dbt compile itself fails because of the name conflict which didn't happen with source and model so i think we are good on this? Please let me know if there is anything else I can do to help you test and merge this. Would've also been nice to be able to run the integration tests but I suppose that would need more changes and I'm not familiar with your CI/CD process.

Reply:
hey @mateuszklimek this has now been improved and it should work with everything since we pull the node from the existing depends_on list